config/toml.go

```diff
@@ -36,20 +36,30 @@ func EnsureRoot(rootDir string) {
 	if err := tmos.EnsureDir(filepath.Join(rootDir, DefaultConfigDirName), DefaultDirPerm); err != nil {
 		panic(err.Error())
 	}
-	tmcfg.EnsureRoot(rootDir)
 
 	configFilePath := filepath.Join(rootDir, DefaultConfigDirName, DefaultConfigFileName)
 
 	// Write default config file if missing.
 	if !tmos.FileExists(configFilePath) {
-		writeDefaultConfigFile(configFilePath)
+		writeDefaultConfigFile(configFilePath, rootDir)
 	}
 }
 
 // XXX: this func should probably be called by cmd/tendermint/commands/init.go
 // alongside the writing of the genesis.json and priv_validator.json
-func writeDefaultConfigFile(configFilePath string) {
-	WriteConfigFile(configFilePath, DefaultConfig())
+func writeDefaultConfigFile(configFilePath, rootDir string) {
+	cfg := DefaultConfig()
+	defaultSLconfig := settlement.Config{
+		KeyringBackend: "test",
+		NodeAddress:    "http://127.0.0.1:36657",
+		RollappID:      "",
+		KeyringHomeDir: rootDir,
```

Comment:
when following the RDK readme.md - the keyring home dir generated by it collides with the default keyring home dir generated by the toml. 

so the readme.md says to generate it under "rootDir/sequencer" and the default dymint.toml has "rootDir", so when running the rollapp using the `start` command it can't find the key.

I suggest prefixing both defaults (i.e in the toml + in the rdk readme.md) with rootDor/hub-keyring for clarity and consistency. 

IMO the directory name `rootDir/sequencer` in the rdk readme.md is confusing (as it's a dir which contains hub keys)